pkgs/development/tools/carto/default.nix

```
@@ -0,0 +1,29 @@
+{ lib
+, buildNpmPackage
+, fetchFromGitHub
+}:
+
+buildNpmPackage rec {
+  pname = "carto";
+  version = "1.2.0";
+
+  src = fetchFromGitHub {
+    owner = "mapbox";
+    repo = "carto";
+    rev = "v${version}";
+    hash = "sha256-TylMgb2EI52uFmVeMJiQltgNCSh6MutFwUjsYC7gfEA=";
+  };
+
+  npmDepsHash = "sha256-8M9hze71bQWhyxcXeI/EOr0SQ+tx8Lb9LfvnGxYYo0A=";
+
+  dontNpmBuild = true;
+
+  meta = {
+    changelog = "https://github.com/mapbox/carto/blob/${src.rev}/CHANGELOG.md";
+    description = "Fast CSS-like map stylesheets";
```

While this is in the GitHub "About" section, it is terrible at describing what this program actually does. I propose we keep the old description:
```suggestion
    description = "Mapnik Stylesheet Compiler";
```